tests/ArrayHelper/GetValueTest.php

```diff
@@ -53,11 +53,6 @@ private function commonDataProviderFromArray(): array
             ['name', 'test'],
             ['noname', null],
             ['noname', 'test', 'test'],
-            [
```

Comment:
Why removing this test?

Reply:
this test line for `getValue()` it need to be adjusted and also it has been moved to `testGetValueByMatcher()`,
 but sorry, I forgot this one also has to be passed to `getValueByPath()` tests